dagshub/upload/wrapper.py

```diff
@@ -24,6 +26,66 @@ def get_default_branch(src_url, owner, reponame, auth):
     return res.json().get('default_branch')
 
 
+def create_repo(repo_name, description="", private=False, auto_init=False, gitignores="", license="", readme="", template="notebook-template"):
+    import logging
+    logging.basicConfig(level=logging.DEBUG)
+    import dagshub.auth
+    from dagshub.auth.token_auth import HTTPBearerAuth
+
+    username = config.username
+    password = config.password
+    if username is not None and password is not None:
+        return username, password
+    try:
+        token = dagshub.auth.get_token(code_input_timeout=0)
+    except dagshub.auth.OauthNonInteractiveShellException:
+        logger.debug("Failed to perform OAuth in a non interactive shell")
```

Comment:
I think you shouldn't catch this error, and just silently fail. Let the user code crash.